.editorconfig

```diff
@@ -271,6 +271,6 @@ csharp_style_var_elsewhere = true:warning
 
 [src/{VisualStudio}/**/*.{cs,vb}]
 # CA1822: Make member static
-# Not enforced as a build 'warning' for 'VisualStudio' layer due to large number of false positives from https://github.com/dotnet/roslyn-analyzers/issues/3857 and https://github.com/dotnet/roslyn-analyzers/issues/3858
-# Additionally, there is a risk of accidentally breaking an internal API that partners rely on though IVT.
-dotnet_diagnostic.CA1822.severity = suggestion
+# There is a risk of accidentally breaking an internal API that partners rely on though IVT.
+dotnet_diagnostic.CA1822.severity = warning
```

Comment:
This looks redundant as it's already specified above, but it's required due to #50577